### DISCUSSION TITLE
Dash Mantine Components

### DISCUSSION THREAD

**Discussion by snehilvj** [mantinedev/mantine#418] [Show]
Hi Everyone 😊

I really like the Mantine Components Library. Its a beautiful library and I especially love the native dark theme support. Thanks @rtivital. 

I'm quite excited to present my first open source project: **Dash Mantine Components**
[Dash](https://dash.plotly.com) is a python framework created by plotly for creating interactive web applications. I have converted the Mantine components into dash components, so now they can also be used in python dash apps.

[Repository](https://github.com/snehilvj/dash-mantine-components)
[PyPI](https://pypi.org/project/dash-mantine-components/)

You can install it via pip:
```pip install dash-mantine-components```

I have a TODO list which includes creating a thorough documentation, and support for dark theme. I intend to maintain this project for the long haul.

This is my first open source project and I hope to learn a lot from the community.

Thanks

```python
import dash_mantine_components as dmc
from dash import Dash, Input, Output

app = Dash(__name__)

app.layout = html.Div(
    [
        dmc.DatePicker(
            id="datepicker", format="dddd, MMMM D, YYYY", style={"width": "250px"}
        ),
        dmc.Space(h=20),
        dmc.Text(id="text"),
        dmc.Button("Click Me!")
    ]
)


@app.callback(Output("text", "children"), Input("datepicker", "date"))
def datepicker(date):
    return date


if __name__ == "__main__":
    app.run_server(debug=True)
```

**Comment by rtivital** [comment]
Thanks for sharing! Feel free to share apps that you've built with dash and Mantine, I've never heard of Dash before, so I'm interested to see how it's done

**Reply by snehilvj** [reply]
Dash is popularly used for creating interactive web apps (analytical dashboards) in banks, academia, etc. where knowledge of python is prevalent but not javascript/react. This library enables the use of Mantine components in python. 

I'll make sure to create and share some good examples of what can be done with dash and mantine.
